ci/funs.sh

```diff
@@ -1,8 +1,101 @@
-# utilities used in CI pipelines to avoid duplication.
+# Utilities used in CI pipelines and tooling to avoid duplication.
+# Prefer nim scripts whenever possible.
+# functions starting with `_` are considered internal, less stable.
 
 echo_run () {
   # echo's a command before running it, which helps understanding logs
   echo ""
   echo "$@"
   "$@"
 }
+
+nimDefineVars(){
+  # D20210421T185609:here
+  nim_csourcesDir=csources # where we clone the latest csources (currently csources_v1.git)
+  nim_csourcesDir_v0=csources_v0 # where we move or clone the old csources.git
```

Comment:
That's confusing, please don't map "csources" to "csources_v1" and then "csources_v0" to "csources".

Reply:
TBH i went back and forth bw this version and a version where i was using csources + csources_v1, so shall i change it to csources + csources_v1 ?

Reply:
Yes.

Reply:
done